eventsourced-nats/src/evt_log.rs

```diff
@@ -373,25 +373,25 @@ fn evt_stream_max_bytes_default() -> i64 {
 
 #[cfg(test)]
 mod tests {
-    use super::*;
-    use crate::tests::NATS_VERSION;
+    use crate::{tests::NATS_VERSION, NatsEvtLog, NatsEvtLogConfig};
     use error_ext::BoxError;
-    use eventsourced::binarize;
-    use futures::TryStreamExt;
-    use std::future;
+    use eventsourced::{binarize, evt_log::EvtLog};
+    use futures::{StreamExt, TryStreamExt};
+    use std::{future, num::NonZeroU64};
     use testcontainers::{clients::Cli, core::WaitFor};
     use testcontainers_modules::testcontainers::GenericImage;
     use uuid::Uuid;
 
     #[tokio::test]
     async fn test_evt_log() -> Result<(), BoxError> {
-        let client = Cli::default();
+        let tc_client = Cli::default();
```

Comment:
nit: maybe just `containers`? Had to think about what `tc_client` was.

Reply:
Yes, I like that.